configuration/conf/config.properties

```diff
@@ -84,11 +84,11 @@ VERIFICATION_STEP_SIZE=10000
 
 ############## 数据：设备、传感器、客户端 ##################
 # 设备总数
-DEVICE_NUMBER=6000
+DEVICE_NUMBER=60
 # 实际写入设备比例，(0, 1]
 REAL_INSERT_RATE=1.0
 # 每个设备的传感器总数
-SENSOR_NUMBER=200
+SENSOR_NUMBER=20
```

Comment:
There are no need to update default config

Reply:
sure